app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardsSettingFragment.kt

```diff
@@ -65,4 +77,19 @@ class CreditCardsSettingFragment : PreferenceFragmentCompat() {
 
         return super.onPreferenceTreeClick(preference)
     }
+
+    override fun onStop() {
+        removeFlagSecure()
+        super.onStop()
+    }
+
+    /**
+     * Checks var isNextDestinationCreditCardEditor to decide whether to remove or keep
+     * WindowManager.LayoutParams.FLAG_SECURE
+     */
+    private fun removeFlagSecure() {
+        if (!isNextDestinationCreditCardEditor) {
```

Comment:
Seeing that `CreditCardEditorFragment` sets `FLAG_SECURE` on it's own (which is probably desired) maybe we don't need this if check and have `onStop` always remove the flag?